### PR TITLE
Add in resources.

### PR DESCRIPTION
**Pull request by hshin-g**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [ ] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [ ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [ ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [ ] My PR has been functionally tested.
- [ ] All of the [unit-tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) still pass.
- [ ] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: _includes/docs/latest/resource-coverage.md [hunk 1]
```diff
@@ -2,16 +2,22 @@
 | Resource                                |
 | --------------------------------------- |
 | [App Engine Apps](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps) |
+| [App Engine Services](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services) |
+| [App Engine Version](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions)
```
Comment (blueandgold) [inline _includes/docs/latest/resource-coverage.md]: Looks like `appengine_instance` is also needed.

https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/services/inventory/base/resources.py#L2261-L2264

File: _includes/docs/latest/resource-coverage.md [hunk 2]
```diff
@@ -2,16 +2,22 @@
...
+| [App Engine Services](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services) |
+| [App Engine Version](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions) 
 | [BigQuery](https://cloud.google.com/bigquery/docs/reference/rest/v2/) |
 | [Billing Accounts](https://cloud.google.com/billing/reference/rest/v1/billingAccounts) |
 | [Buckets](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) |
 | [Bucket Access Controls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) |
 | [Cloud APIs Configuration](https://cloud.google.com/apis/) |
 | [Cloud Audit Logging Configuration](https://cloud.google.com/iap/docs/audit-log-howto) |
-| [Cloud Dataproc Clusters](https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters) |
+| [Cloud Dataproc Clusters](https://cloud.google.com/dataproc/docs/guides/create-cluster) |
```
Comment (blueandgold) [inline _includes/docs/latest/resource-coverage.md]: We should try to point this to the resource representation (i.e. the json).

File: _includes/docs/latest/resource-coverage.md [hunk 3]
```diff
@@ -49,8 +56,11 @@
 | [G Suite Groups](https://developers.google.com/admin-sdk/directory/v1/reference/groups) |
 | [G Suite Group Members](https://developers.google.com/admin-sdk/directory/v1/reference/members) |
 | [G Suite Users](https://developers.google.com/admin-sdk/directory/v1/reference/users) |
+| [Kubernetes Engine Clusters](https://cloud.google.com/kubernetes-engine/docs/reference/rest/) |
 | [Kubernetes Engine Configurations](https://cloud.google.com/kubernetes-engine/docs/reference/rest) |
 | [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) |
+| [Logging Organization Sinks](https://cloud.google.com/logging/docs/reference/v2/rest/v2/organizations.sinks) |
+| [Logging Folder Sinks](https://cloud.google.com/logging/docs/reference/v2/rest/v2/folders.sinks) |
```
Comment (blueandgold) [inline _includes/docs/latest/resource-coverage.md]: Should we also have one for project?

Comment (hshin-g) [reply _includes/docs/latest/resource-coverage.md]: Added in.

File: _includes/docs/latest/resource-coverage.md (hunk 2 above)
Comment (hshin-g) [reply _includes/docs/latest/resource-coverage.md]: Fixed.

File: _includes/docs/latest/resource-coverage.md (hunk 1 above)
Comment (hshin-g) [reply _includes/docs/latest/resource-coverage.md]: Added in.

**Pull request merged**